packages/wp-now/src/wp-now.ts

```diff
@@ -361,12 +362,20 @@ function mountMuPlugins(php: NodePHP, vfsDocumentRoot: string) {
 	);
 }
 
-function mountSqlitePlugin(php: NodePHP, vfsDocumentRoot: string) {
-	const sqlitePluginPath = `${vfsDocumentRoot}/wp-content/plugins/${SQLITE_FILENAME}`;
+function mountSqlitePlugin(
+	php: NodePHP,
+	vfsDocumentRoot: string,
+	wpContentPath: string
+) {
+	const sqlitePluginPath = path.join(
+		wpContentPath,
+		'plugins',
+		SQLITE_FILENAME
+	);
 	if (php.listFiles(sqlitePluginPath).length === 0) {
-		php.mount(getSqlitePath(), sqlitePluginPath);
+		fs.copySync(getSqlitePath(), sqlitePluginPath);
 		php.mount(
-			path.join(getSqlitePath(), 'db.copy'),
+			path.join(sqlitePluginPath, 'db.copy'),
 			`${vfsDocumentRoot}/wp-content/db.php`
 		);
```

Comment:
I think it's best to move this plugin to mu-plugins. This will prevent unwanted updates and will match what hosting platforms do.

Reply:
I will keep it as is for now and attempt to fix plugin installing.

Reply:
> I think it's best to move this plugin to mu-plugins. This will prevent unwanted updates and will match what hosting platforms do.

This is what Playground core does, too, to support multisites. Enabling a multisite requires deactivating all plugins, and this is one plugin we cannot deactivate.

Reply:
When I try mounting the plugin to mu-plugins the [file_exists check fails.](https://github.com/WordPress/sqlite-database-integration/blob/8a4efbecf652a2ee3c5facd31ba0542b0d397c13/db.copy#L17) This prevents the DB from loading.

I assume that the root cause of this is the same as the [is dir issue with updates](https://github.com/WordPress/playground-tools/issues/134#issuecomment-1973182771).

```
function mountSqlitePlugin(php: NodePHP, vfsDocumentRoot: string) {
	const sqlitePluginPath = `${vfsDocumentRoot}/wp-content/mu-plugins/${SQLITE_FILENAME}`;
	if (php.listFiles(sqlitePluginPath).length === 0) {
		php.mount(getSqlitePath(), sqlitePluginPath);
		php.writeFile(
			`${vfsDocumentRoot}/wp-content/db.php`,
			php
				.readFileAsText(path.join(getSqlitePath(), 'db.copy'))
				.replace(
					'{SQLITE_IMPLEMENTATION_FOLDER_PATH}',
					sqlitePluginPath
				)
		);
	}
}
```

Reply:
We could copy the files, but in that case we would also need to copy the other mu-plugin files.

Reply:
This is how Playground does it:

https://github.com/WordPress/wordpress-playground/blob/8b74852e9701f5083b367f9a294f34200230ce79/packages/playground/wordpress/build/Dockerfile#L36-L42

Perhaps something from that script is missing in the wp-now logic? Ideally, both tools will soon be covered by the PHP Blueprints library.